lib/action_kit.rb

```diff
@@ -0,0 +1,3 @@
+module ActionKit
+end
+
```

Comment:
also why does this stuff go in lib and not app/services? what is our mental dividing line?